packages/next/src/views/Version/RenderFieldsToDiff/index.tsx

```diff
@@ -123,7 +123,7 @@ const RenderFieldsToDiff: React.FC<Props> = ({
           // At this point, we are dealing with a `row`, etc
           if ('fields' in field) {
             return (
-              <Nested
+              <Component
```

Comment:
Not sure if there was a reason for the Nested component to be "fixed" here. I think we can just render the mapped component here, assuming that every nested field has a mapped component. Needed this to render the Nested or Collapsible component depending on the field.